azure-yaml-container-apps/Pulumi.yaml

```diff
@@ -74,7 +74,7 @@ resources:
     properties:
       resourceGroupName: ${resourceGroup.name}
       kubeEnvironmentId: ${kubeEnv}
-      configuration:
+      config:
```

Comment:
Regexp based replace would benefit from `^configuration:` to prevent this kind of thing.